### PR TITLE
Keep query string when force secure redirect

### PR DESCRIPTION
**Pull request by ivancrneto**
## Test

- Mark a url as force secure
- Visit it with parameters, like: http://localhost/test.html?review=1088923#test-hash
- You will be redirected to  https://localhost/test.html?review=1088923#test-hash instead of  http://localhost/test.html

### CODE REVIEW

File: urlographer/views.py [hunk 1]
```diff
@@ -74,7 +79,8 @@ def route(request):
     request.urlmap = url
 
     if url.force_secure and not request.is_secure():
-        response = HttpResponsePermanentRedirect(unicode(url))
+        url_to = get_redirect_url_with_query_string(request, unicode(url))
+        response = HttpResponsePermanentRedirect(url_to)
```
Comment (jvzammit) [inline urlographer/views.py]: Should use `redirect` function here instead of instantiating the class, and pass `status_code` keyword arg.

**Comment by jvzammit** [comment]
Practical test with query parameters on URLMap with `force_secure` enabled redirected while retaining query parameters. Small code suggestion as indicated @ivancrneto , once you submit feedback on that, then this is good to merge.

File: urlographer/views.py (hunk 1 above)
Comment (ivancrneto) [inline urlographer/views.py]: I was thinking about that but I chose to follow the patterns of the lines after. If I changed that I would have to change all and I was afraid to introduce more bugs

Comment (jvzammit) [inline urlographer/views.py]: That sort of usage causes a warning; that's why I suggested to user `redirect`. Anyway we can have an additional ticket to address that then? And move on with this?

**Comment by jvzammit** [comment]
@ivancrneto Merging.

File: urlographer/views.py (hunk 1 above)
Comment (ivancrneto) [inline urlographer/views.py]: Sure let's have a ticket to change that along with the changes needed for django 1.6. Thanks for merging.

Comment (jvzammit) [inline urlographer/views.py]: Added the ticket as an issue on django-urlographer once I merged, FYI.

**Pull request merged**